examples/go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/gnolang/gno/examples
```

Comment:
is this file needed?

Reply:
Right now yes, because of this -> https://github.com/gnolang/gno/pull/126/files/e41af05ad59b5ac45a8d192fdda32c52821a72bc#diff-cd249c062d77b4d6a5621756423850d0708ffbe4aae554ce33cdc5b908b0ee39R25

My goal is to make it unnecessary to have any go.mod, but I think that it will requires some adjustments and to merge all the precompile stuff to be able to use go mod’s cache effectively

FYI, I started a standalone repo to experience by myself Gno smart contracts from an external developer; my goal is to make it simpler over tie

Reply:
the link is broken